web/src/features/hfiCalculator/components/DailyViewTable.tsx

```diff
@@ -50,102 +59,182 @@ const useStyles = makeStyles({
-                                      className={classes.station}
-                                      key={`station-${stationCode}`}
-                                    >
-                                      <TableCell key={`station-${station.code}-name`}>
-                                        {station.station_props.name} ({station.code})
-                                      </TableCell>
-                                      <TableCell
-                                        key={`station-${station.code}-elevation`}
-                                      >
-                                        {station.station_props.elevation}
-                                      </TableCell>
-                                      <TableCell
-                                        key={`station-${station.code}-fuel-type`}
-                                      >
-                                        {station.station_props.fuel_type.abbrev}
-                                      </TableCell>
-                                    </TableRow>
-                                  )
-                                }
-                              )}
-                            </React.Fragment>
-                          )
-                        })}
-                      </React.Fragment>
-                    )
-                  })}
-                </TableBody>
-              </Table>
-            </TableContainer>
-          </Paper>
-        </div>
-      )}
+      <div className={classes.controls}>
+        <Typography component="div" variant="subtitle2">
+          {props.title}
+        </Typography>
+        <Button color="primary" onClick={props.previousDay}>
+          Previous
+        </Button>
+        {props.currentDay}
+        <Button color="primary" onClick={props.nextDay}>
+          Next
+        </Button>
+      </div>
+      <Paper className={classes.paper} elevation={1}>
+        <TableContainer className={classes.tableContainer}>
+          <Table stickyHeader aria-label="daily table view of HFI by planning area">
+            <TableHead>
+              <TableRow>
+                <TableCell key="header-location">Location</TableCell>
+                <TableCell key="header-elevation">
+                  Elev.
+                  <br />
+                  (m)
+                </TableCell>
+                <TableCell key="header-fuel-type">
+                  FBP
+                  <br />
+                  Fuel
+                  <br />
+                  Type
+                </TableCell>
+                <TableCell>Status</TableCell>
+                <TableCell>
+                  Temp
+                  <br />
+                  (°C)
+                </TableCell>
+                <TableCell>
+                  RH
+                  <br />
+                  (%)
+                </TableCell>
+                <TableCell>
+                  Wind
+                  <br />
+                  Dir
```

Comment:
add units of measure for this column?

Reply:
I'm not sure what that would be for wind direction. Is it the same as temperature?